packages/docs/src/stories/templates/switch.stories.ts

```diff
@@ -0,0 +1,54 @@
+import '../../../../components/src/solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev', 'autodocs'],
+  title: 'Templates/Switch',
+  parameters: {
+    chromatic: { disableSnapshot: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=15034-649&t=obZU2ca0L5CjCDA4-4'
+    }
+  }
+};
+
+/**
+ * ### Switch with Tooltip
+ */
+export const Default = {
```

Comment:
Please make the canvas height bigger so that when you open a select you can see the options